Web/RotaMe.Web/Areas/Owner/Views/Projects/RemoveUser.cshtml

```diff
@@ -0,0 +1,82 @@
+﻿@model RotaMe.Web.InputModels.Owner.Projects.UserRemoveFromProjectInputModel
+@using RotaMe.Web.ViewModels.Owner.Projects
+@addTagHelper *, Microsoft.AspNetCore.Mvc.TagHelpers
+
+@{
+    ViewData["Title"] = "Remove User";
+    Layout = "~/Views/Shared/_Layout.cshtml";
+}
+
+@section Styles{
+
+    <!-- ekko-lightbox css -->
+    <link rel="stylesheet" href="~/assets/css/plugins/select2.min.css">
+}
+
+<!-- [ Main Content ] start -->
+<section class="pcoded-main-container">
+    <div class="pcoded-content">
+
+        <partial name="~/Views/Shared/_BreadcrumbPartial.cshtml" model='new BreadcrumbPartialViewModel() { Controller = "Projects", Action = "Remove User from Project"}' />
+        <!-- [ Main Content ] start -->
+        <div class="row">
+            <div class="col-sm-12">
+                <div class="card">
+                    <div class="card-header">
+                        <h5>Remove User from Project</h5>
+                    </div>
+                    <div class="card-body">
+                        <form method="post">
+                            <div class="row">
+                                <div class="col-xl-5 col-md-4 mb-5">
+                                    <h5>Select Project</h5>
+                                    <hr>
+                                    <select asp-for="ProjectId" class="js-example-basic-single form-control project-remove-user-selector">
+                                        @foreach (var project in (IEnumerable<ProjectsListToRemoveUserViewModel>)this.ViewData["projects"])
+                                        {
+                                            if (project.Selected)
+                                            {
+                                                <option selected value="@project.Id" data-users="@project.Users">@project.Title</option>
+                                            }
+                                            else
+                                            {
+                                                <option value="@project.Id" data-users="@project.Users">@project.Title</option>
+                                            }
```

Comment:
can that be 
`<option @if(project.Selected) selected @endif value="@project.Id" data-users="@project.Users">@project.Title</option>`

Reply:
it's not possible to write "if" there

Reply:
thats doubtful.
what about https://stackoverflow.com/questions/4770605/how-to-use-if-statements-with-razor-and-inline-code-blocks